src/main/groovy/com/profitbricks/sdk/model/Request.groovy

```diff
@@ -0,0 +1,36 @@
+package com.profitbricks.sdk.model
+
+import com.profitbricks.sdk.annotation.*
+import groovy.transform.*
+
+/**
+ * a Request POGO
+ * see: https://devops.profitbricks.com/api/cloud/v3/#lans
+ *
+ * Created by ali bazlamit on 30/03/17.
+ * Copyright (c) 2017, ProfitBricks GmbH
+ */
+@ToString(includeNames = true, ignoreNulls = true, includeSuperProperties = true, includePackage = false, excludes = ['resource'])
+@EqualsAndHashCode(callSuper = true)
+final class Request extends ModelBase {
```

Comment:
what would be the use of this POGO? also, as its not really deserialized its usefulness escapes me (also note, this SDK handles this transparently, for reasons)